mpas_analysis/ocean/time_series_sst.py

```diff
@@ -252,11 +252,25 @@ def run_task(self):  # {{{
                 lineWidths.append(1.5)
                 legendText.append(preprocessedReferenceRunName)
 
+            if config.has_option(self.taskName, 'firstYearXTicks'):
+                firstYearXTicks = config.getint(self.taskName,
+                                                'firstYearXTicks')
+            else:
+                firstYearXTicks = None
+
+            if config.has_option(self.taskName, 'yearStrideXTicks'):
+                yearStrideXTicks = config.getint(self.taskName,
+                                                 'yearStrideXTicks')
+            else:
+                yearStrideXTicks = None
```

Comment:
`yearStrideXTicks` is `None` if it's not in the config file, and the same is true for other analysis tasks.